### PR TITLE
add table of contents to IMPLEMENTATION_COVERAGE.md

### PR DESCRIPTION
**Pull request by shiba24**

### CODE REVIEW

**Comment by shiba24** [comment]
@mikegrima

File: IMPLEMENTATION_COVERAGE.md [hunk 1]
```diff
@@ -1,3 +1,139 @@
...
+- [codebuild - 0% implemented](#codebuild---0--implemented)
+- [codecommit - 0% implemented](#codecommit---0--implemented)
+- [codedeploy - 0% implemented](#codedeploy---0--implemented)
+- [codepipeline - 0% implemented](#codepipeline---0--implemented)
+- [codestar - 0% implemented](#codestar---0--implemented)
+- [cognito-identity - 0% implemented](#cognito-identity---0--implemented)
+- [cognito-idp - 34% implemented](#cognito-idp---34--implemented)
+- [cognito-sync - 0% implemented](#cognito-sync---0--implemented)
+- [comprehend - 0% implemented](#comprehend---0--implemented)
+- [config - 0% implemented](#config---0--implemented)
```
Comment (mikegrima) [inline IMPLEMENTATION_COVERAGE.md]: Yours truly added a bunch of AWS Config support. A lot of the data here is not accurate.

**Comment by jallen182** [comment]
Would it be more readable if we turned these into collapsible dropdowns?
<details>
    <summary> implemented methods (38%) </summary>
     ...List of methods....
</details>

**Comment by bblommers** [comment]
By now the implementation coverage has collapsible dropdowns, and has a separation between implemented services and services without any implementation - making it much more readable.
I'll close this, but if there are other ways to make the docs more readable, suggestions are always welcome!

**Pull request closed without merging**